internal/performance/api_util_test.go

```diff
@@ -0,0 +1,42 @@
+package performance
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/smartcontractkit/chainlink/internal/cltest"
+	"fmt"
+	"github.com/smartcontractkit/chainlink/store/models"
+)
+
+func TestGetSchemas(t *testing.T) {
+	schemas := GetSchemas()
+	assert.True(t, len(schemas) == 2, "No schemas were returned from file")
+}
+
+func TestGetBasicAuthHeader(t *testing.T) {
+	app, cleanup := cltest.NewApplicationWithKeyStore()
+	defer cleanup()
+
+	header := GetBasicAuthHeader(app.Store)
+	assert.True(t, len(header.Get("Authorization")) > 0, "Authorization header wasn't set")
+}
+
+func TestTargets(t *testing.T) {
+	app, cleanup := cltest.NewApplicationWithKeyStore()
+	defer cleanup()
+
+	targets := GetCreateJobTargets(app)
+	assert.Equal(t, targets[0].URL, fmt.Sprintf("%s/v2/jobs", app.Server.URL))
```

Comment:
You'll probably need to rebase, and update the routes to `/specs` in place of `/jobs` everywhere. We received some feedback that it was unclear so we renamed it.